x-pack/plugins/index_management/__jest__/client_integration/index_details_page/index_details_page.test.ts

```diff
@@ -58,4 +93,24 @@ describe('<IndexDetailsPage />', () => {
     const tabContent = testBed.actions.getActiveTabContent();
     expect(tabContent).toEqual('Pipelines');
   });
+
+  it('navigates back to indices', async () => {
+    jest.spyOn(testBed.routerMock.history, 'push');
+    await testBed.actions.clickBackToIndicesButton();
+    expect(testBed.routerMock.history.push).toHaveBeenCalledTimes(1);
+    expect(testBed.routerMock.history.push).toHaveBeenCalledWith('/indices');
+  });
+
+  it('renders a link to discover', () => {
```

Comment:
only testing that the button is displayed, since the discover link component has its own tests